internal/server/server.go

```diff
@@ -495,7 +495,12 @@ func (s *Server) syncIdentityInfo(ctx context.Context, tx *data.Transaction, ide
 		}
 
 		// update current identity provider groups and account status
+		t := time.Now()
 		_, err = data.SyncProviderUser(ctx, tx, providerUser, oidc)
+		outboundRequestDuration.With(prometheus.Labels{
```

Comment:
This ends up tracking a bunch of operations , including DB queries, right?

We probably want this metric to be include the time for only the outbound HTTP request, not this high level operation.

Reply:
This is measurement is too broad as @dnephin mentioned. Instead, it should measure the requests sent by the OIDC client, specifically `oidcClient.RefreshAccessToken()` and `oidcClient.GetUserInfo()` in `data.SyncProviderUser()` with each measurement having a distinct label value